frontend/tailwind.config.js

```diff
@@ -1,21 +1,7 @@
 module.exports = {
+	mode: 'jit',
```

Comment:
Seems the indentation here is inconsistent - would it be possible to get a basic prettier config set up to auto-format everything?

Reply:
done